libnetwork/iptables/firewalld.go

```diff
@@ -108,8 +129,10 @@ func signalHandler() {
 	for signal := range connection.signal {
 		switch {
 		case strings.Contains(signal.Name, "NameOwnerChanged"):
-			firewalldRunning = checkRunning()
+			ctx, cancel := context.WithTimeout(context.Background(), dbusConnectionTimeout)
+			firewalldReady, _ = checkRunning(ctx)
```

Comment:
At a quick glance, this is the only place we call `checkRunning()`, and we don't use the error-return. We should either remove the return, or handle the error (which could be "logging" if we don't really care)?